python/lsst/ts/audiotrigger/schemas/heartbeat.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Which code is actually using this schema? Same for the other two. The schemas look good but are only useful if they are actually used as well. I don't mean to have them removed but they should be used somewhere.